### PR TITLE
Bitly docs

### PR DESCRIPTION
**Pull request by Shraddha2104**
Hi @tanay1337 ,
✨ I have added basic instructions for Bitly
Let me know if any changes are needed!

### CODE REVIEW

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- Link
+	- Create a link
+	- Get a link
+	- Update a link
+
+
+## Example Usage
+
+This workflow shows you how to create a new link. You can also find the [workflow](https://n8n.io/workflows/442) on this website. This example usage workflow uses the following two nodes.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:23]: Could you change it to "on the website"?

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+- [Start](../../core-nodes/Start)
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:25]: Could you make sure it's similar to other nodes' documentation?

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+permalink: /nodes/n8n-nodes-base.bitly
+---
+
+# Bitly
+
+[Bitly](https://bitly.com/) is URL shortening service and a link management platform that allows users to shorten, create and share trusted, powerful links for businesses.
+
+::: tip 🔑 Credentials
+The instructions for authenticating with Bitly are [here](../../../credentials/Bitly).
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:10]: Could you make sure it's similar to other nodes' documentation?

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+
+![A workflow with the Bitly node](./workflow.png)
+
+### 1. Start node
+
+The start node exists by default when you create a new workflow.
+
+### 2. Bitly node
+
+1. First, ensure that you have entered authentication credentials through [this tutorial](../../../credentials/Bitly/README.md).
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:38]: Could you make sure it's similar to other nodes' documentation?

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+2. To create a link, select the *Link* option in the *Resource* field.
+3. Select the *Create* option in the *Operation* field.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:40]: These are default options for this node, could you remove them?

File: docs/nodes/nodes-library/nodes/Bitly/README.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+4. Enter the long URL in the *Long URL* field.
```
Comment (smamudhan) [inline docs/nodes/nodes-library/nodes/Bitly/README.md:41]: Could you change this to: `Enter the URL in the *Long URL* field.`?

**Comment by Shraddha2104** [comment]
> Hello @Shraddha2104, thank you for contributing!
> Could you make the following changes before we merge into master?

Hey @smamudhan 
I have updated the docs accordingly. Let me know if any changes are needed!

**Comment by tanay1337** [comment]
Merged, thanks a lot @Shraddha2104 🙌

**Pull request merged**